src/RulesetEngine.php

```diff
@@ -45,4 +46,23 @@ public function getViolations(Result $dependencyResult, ClassNameLayerResolverIn
 
         return $violations;
     }
+
+    /**
+     * Get skipped violations
```

Comment:
This comment brings no added value, too.

Reply:
fixed